modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CSharpClientCodegen.java

```diff
@@ -253,6 +264,17 @@ public void setOptionalAssemblyInfoFlag(boolean flag) {
         this.optionalAssemblyInfoFlag = flag;
     }
 
+    @Override
+    public CodegenModel fromModel(String name, Model model, Map<String, Model> allDefinitions) {
+        CodegenModel codegenModel = super.fromModel(name, model, allDefinitions);
+     if (allDefinitions != null && codegenModel != null && codegenModel.parent != null && codegenModel.hasEnums) {
```

Comment:
For your next PR, would be good if you can ident the code following the Java style guide: https://google.github.io/styleguide/javaguide.html